src/raylib.h

```diff
@@ -692,7 +700,7 @@ extern "C" {            // Prevents name mangling of functions
 //------------------------------------------------------------------------------------
 
 // Window-related functions
-RLAPI void InitWindow(int width, int height, void *data);         // Initialize window and OpenGL context
+RLAPI void InitWindow(int width, int height, windowTitle name);  // Initialize window and OpenGL context
```

Comment:
Hey @Martinfx, many thanks for the effort on this review but for this time I prefer to leave it a it was, using `void *data` and avoiding any `PLATFORM_` define in `raylib.h`.

Reply:
or rewrite as https://github.com/gfxprim/gfxprim/blob/master/demos/c_simple/backend_example.c
https://github.com/gfxprim/gfxprim/tree/master/include/backends

Reply:
I know, but all possible solutions make the code more complex than leaving it as it is...

Reply:
I didn't dig into the code, why couldn't the platform specific data be stored to a raylib internal `static` buffer?

Reply:
Ah, never mind. I thought it was just an output parameter.

Reply:
For android, we could define `android_main` in raylib:
```
void android_main(struct android_app* app) {
    InitAndroidApp(app);
    exit(main(1, &(char[]){ "raylib", NULL }));
}
```
And allow use of `InitWindow(.., "windowTitle)` as usual. This is beneficial to users because they don't need #ifdefs in their code. If they require access to the `struct android_app`, we could add a `GetAndroidState` function. Would that be ok? I haven't tried raylib on Android.

Maybe a similar arrangement can be found for UWP, but I haven't seen UWP example code.

Reply:
Ok

Reply:
Hi @a3f! That would be amazing if we could avoid forcing user to `#ifdef PLATFORM_ANDROID` on their code! Sorry, but I don't understand exactly your proposal... that code would be defined in `core.c`?

Reply:
@raysan5 See #508.

Reply:
@a3f @Martinfx #508 solved this issue `InitWindow()` in Android platforms, maybe we could find a similar approach for UWP and then let `void *data` to be just `const char *title`.

UWP example is provided as a [VS2015 project](https://github.com/raysan5/raylib/tree/master/project/vs2015.UWP) and [here it is initialization point](https://github.com/raysan5/raylib/blob/master/project/vs2015.UWP/raylib.App.UWP/App.cpp#L425).